File: Exercicio Python/Exercicio_5.py
def num(numero):
    numero[i] = int(input("Digite um valor:"))

def par(numero):
    if(numero[i] % 2 == 0):
        print(numero[i])

numero = [0] * 5

for i in range(0,len(numero)):
    num(numero)
print('Os numeros pares são:')
for i in range(0,len(numero)):
    par(numero)

File: Exercicios C/Semana 5/Exercicio_5.cpp
#include <stdio.h>

int main(){
	
	int a, b;
    int Numeros[30];
    
	for (a = 0;a<30;a++){
    	printf("Digite um numero: ");
    	scanf("%d", &Numeros[a]);
    	fflush(stdin);
	}
    
	for (b = 29; b >= 0; b--){
        printf("%d\n", Numeros[b]);
    }
    return 0;
}
